hack/README.md

```diff
@@ -76,31 +78,85 @@ These may be passed to a target as `$ make VARIABLE=value target`
 - `DOCKER_TAG`: (default: latest) Set global version tags for image and manifest creation
 - `VERBOSITY`: (default: 1) Set global log level verbosity
 - `PULL_POLICY`: (default: IfNotPresent) Set global CDI pull policy
-- `TEST_ARGS`: A variable containing a list of additional ginkgo flags to be passed to functional tests. The string "--test-args=" must prefix the variable value. For example: `make TEST_ARGS="--test-args=-ginkgo.noColor=true" test-functional >& foo`
+- `TEST_ARGS`: A variable containing a list of additional ginkgo flags to be passed to functional tests. The string "--test-args=" must prefix the variable value. For example:
+
+             `make TEST_ARGS="--test-args=-ginkgo.noColor=true" test-functional >& foo`.
+
+  Note: the following extra flags are not supported in TEST_ARGS:
+     -master
+     -cdi-namespace
+     -kubeconfig
+     -kubectl-path
```

Comment:
When viewed formatted this list of variables is a single list with no commas, etc. I think it should be a bullet-ed list or just a comma separated sentence.